integration/lib.go

```diff
@@ -121,3 +136,64 @@ func CompareDir(expected, actual string, replacements map[string]string) (bool,
 
 	return true, nil
 }
+
+func prettyAndCleanJSON(data []byte, keysToIgnore []string) []byte {
```

Comment:
do we still want this and `modifyField` ?

Reply:
Removed `modifyField`. Left this pulled out of the `CompareDir` for now, it was repeated, so /shrug. It's probably better to be out?